google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -27,7 +35,44 @@ public static CloudSdkService getInstance() {
     return ServiceManager.getService(CloudSdkService.class);
   }
 
+  @Nullable
   public abstract String getCloudSdkHomePath();
 
   public abstract void setCloudSdkHomePath(String path);
+
```

Comment:
a lot of this kind of stuff does kind of feel like it could be passed straight through to the common lib. Wrapping it as an IJ service isn't terrible though, but then the name of the service seems a bit off.